feg/cloud/go/services/feg_relay/gw_to_feg_relay/servicers/relay_router.go

```diff
@@ -28,7 +28,6 @@ const (
 	FegS6aProxy     gateway_registry.GwServiceType = "s6a_proxy"
 	FegSessionProxy gateway_registry.GwServiceType = "session_proxy"
 	FegHello        gateway_registry.GwServiceType = "feg_hello"
-	FegSwxProxy     gateway_registry.GwServiceType = "swx_proxy"
```

Comment:
Changes not required

Reply:
moved the FegSwxProxy into southbound folder as /southbound/relay_router.go